src/components/Card.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { SyntheticEvent, useMemo } from 'react';
+import { IconButton } from 'react-file-utils';
+import { sanitizeURL } from '../utils';
+
+import { AvatarIcon, CloseIcon } from './Icons';
+
+// export type Props = {|
+// 	image?: ?string,
+// 	images?: Array<{ image: string }>,
+// 	alt?: ?string,
+// 	title?: ?string,
+// 	url?: string,
+// 	nolink?: boolean,
+// 	description?: ?string,
+// 	handleClose?: (e: SyntheticEvent<>) => mixed,
+//  |};
+
+type CardProps = {
+  alt?: string;
+  description?: string;
+  handleClose?: (e: SyntheticEvent) => void;
+  image?: string | null;
+  images?: Array<{ image: string }>;
+  nolink?: boolean;
+  title?: string;
+  url?: string;
+};
+
+const trimUrl = (url?: string) =>
+  url
+    ?.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')
+    .split('/')
+    .shift();
```

Comment:
worth adding some tests to make sure the behavior is the same as old function

Reply:
`
const trimUrl = (url) => {
  let trimmedUrl;
  if (url !== undefined || url !== null) {
    trimmedUrl = url.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '').split('/')[0];
  }
  return trimmedUrl;
};
`

Feeding this bad boy `null` would resolve in crash since JS evaluates first half of the condition and if it's true, it doesn't need to evaluate the second half and goes about its day trying to call `replace` on `null`. But creating a test with truthy values only wouldn't be a bad idea. I'll move it to `utils.js` then.

Reply:
Added test and it seems to be behaving as expected.